TheFirstHomeTask.playground/Contents.swift

```diff
@@ -1,3 +1,35 @@
-import UIKit
+// Sum of separately whole and separately fractional parts of numbers
+var a = 2.5
+var b = 9.7
+var c = 6.9
+var d = 8.2
 
-var greeting = "Hello, playground"
+var arrayOfValues = [a,b,c,d]
+
+var sumOfFractional: Double = 0
+var sumOfWhole: Double      = 0
+
+for i in arrayOfValues {
```

Comment:
We can also rename `i` to something clearer because when you read `i.rounded(.towardZero)` later it's not very clear what is `i`